pkg/build/inline.go

```diff
@@ -205,11 +220,11 @@ func (n *Inliner) objectFiles(ctx context.Context, objFiles []bundle.File, ref b
 		if err != nil {
 			return nil, nil, err
 		}
-		cf.URL = n.PathRewriter.MakeAbs(componentPath, furl).String()
+		cf.URL = makeAbsWithParent(componentPath, furl).String()
 
 		contents, err := n.readFile(ctx, cf)
 		if err != nil {
-			return nil, nil, fmt.Errorf("error reading file %v for component %v: %v", cf, ref, err)
+			return nil, nil, fmt.Errorf("reading file %v for component %v: %v", cf, ref, err)
```

Comment:
I think the old form (with "error reading...") was more go.

Reply:
Done